tasks/eth/011-proto-ver+ownership-transfers/SignFromJson.s.sol

```diff
@@ -94,6 +89,8 @@ contract SignFromJson is OriginalSignFromJson {
         addressesJson = string(vm.ffi(inputs));
 
         _proxies.ProtocolVersions = stdJson.readAddress(addressesJson, "$.protocol_versions_addr");
-        _proxies.SuperchainConfig = stdJson.readAddress(addressesJson, "$.superchain_config_addr");
+
+        addressesJson = vm.readFile("lib/superchain-registry/superchain/extra/addresses/mainnet/op.json");
```

Comment:
we could use `l1ChainName` and `l2ChainName` here with `string.concat`.

Reply:
Addressed you comment anyways: 3e25b352c8e6177d3291a63c365ed6c7c3cae800